front/app/containers/Admin/inspirationHub/Filters/Topic.tsx

```diff
@@ -19,20 +19,12 @@ const Topic = () => {
   const { data: topics } = useProjectLibraryTopics();
 
   const options = topics?.data.map(({ id, attributes: { l1, l2 } }) => {
-    const l1Words = l1.split(' ');
-    const l1FirstWordLower = l1Words[0].toLowerCase();
-    const l1RestWords = l1Words.slice(1);
-    const l1AllWordsNoSpaces = [l1FirstWordLower, ...l1RestWords].join('');
-    const l2NoSpacesCapitalized = l2
-      .split(' ')
-      .map((word) => word.charAt(0).toUpperCase() + word.slice(1))
-      .join('');
-
-    const messageKey = `${l1AllWordsNoSpaces}${l2NoSpacesCapitalized}`;
+    const topicKey = `${l1}: ${l2}`;
+    const message = messages[topicKey];
 
     return {
       value: id,
-      text: formatMessage(messages[messageKey]),
+      text: message ? formatMessage(message) : topicKey,
```

Comment:
This way, in case we missed any topic, we still have the English fallback.

Reply:
Yeah, much simpler. Thanks!